packages/itwin/tree-widget/src/components/trees/VisibilityTreeRenderer.tsx

```diff
@@ -14,13 +14,44 @@ import type { NodeCheckboxRenderProps } from "@itwin/core-react";
 import type { IPresentationTreeDataProvider } from "@itwin/presentation-components";
 import type { VisibilityTreeFilterInfo } from "./Common";
 
+/**
+ * Props for visibility tree node renderer.
+ * @public
+ */
+export interface visibilityTreeRendererProps {
```

Comment:
Classes & interfaces should start in upper case

Reply:
Fixed